test/libsolidity/smtCheckerTests/functions/functions_external_2.sol

```diff
@@ -17,5 +17,3 @@ contract C
 	}
 }
 // ----
-// Warning 6328: (297-321): CHC: Assertion violation might happen here.
-// Warning 4661: (297-321): BMC: Assertion violation happens here.
```

Comment:
CHC can now prove the last assert. Should we change this example to allow changes to the `map`?

Reply:
Yea, I think so. Maybe allow changes only for `map[0]` and `map[1]`

Reply:
Updated.